src/InputWithLabel.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useRef } from 'react';
+
+const InputWithLabel = ({ todoTitle, handleTitleChange, children }) => {
+  const inputRef = useRef();
+  useEffect(() => {
+    inputRef.current.focus();
+  });
```

Comment:
I know that instructions are telling you not to have a dependency array. For small applications like this, it's okay to do this. For the most part you should always add a dependency array in your useEffect. That way the useEffect does not trigger on every re-render.

No change needed, just wanted to point that out.

Reply:
Got it thank you!